doc/tutorial.txt

```diff
@@ -66,7 +66,7 @@ $ tree -d
 
 the configure step becomes
 
-./configure --host=aarch64-linux-gnu LDFLAGS="--static -L./../json-c/." CFLAGS="-I./.." --with-deadline
+./configure --host=aarch64-linux-gnu LDFLAGS="-static -L"`pwd`"/../json-c" CFLAGS="-I./../json-c" --with-deadline
```

Comment:
Mmm, it's working here only if I keep `--static` and `-L"`pwd`"/json-c"`, basically

```
./configure --host=aarch64-linux-gnu LDFLAGS="--static -L"`pwd`"/json-c" CFLAGS="-I./../json-c/" --with-deadline
```

Reply:
according to gcc docs (https://gcc.gnu.org/onlinedocs/gcc/Link-Options.html) , it's `-static`, not `--static`, so i took a look and came to the conclusion that actually both `-static` and `--static` are wrong. proposed solution at the end, keep reading if you're interested.

diffing the make invocations:
```
-/bin/sh ../libtool  --tag=CC   --mode=link gcc  -I./../json-c  -static [...]
-libtool: link: gcc -I./../json-c -o rt-app [...]
+/bin/sh ../libtool  --tag=CC   --mode=link gcc  -I./../json-c  --static [...]
+libtool: link: gcc -I./../json-c --static -o rt-app [...]
```

notice that in the `-static` case, the flag is missing from the linker invocation and in the `--static` case it is passed through (and gcc doesn't complain)

looking at libtool (https://www.sourceware.org/autobook/autobook/autobook_59.html), it is stripping `-static`, which is a flag that it consumes itself:
```‘-static’
    This switch is similar to ‘-all-static’, except that it applies to only the uninstalled Libtool libraries in the build tree. Where possible the static archive from these libraries is used, but the default linking mode is used for libraries which are already installed. ```

the equivalent of gcc's linker invocation with `-static` seems to be `-all-static`:
```‘-all-static’
    Always choose static libraries where possible, and try to create a completely statically linked executable.```

(when i earlier used `-static`, i ended up with a dynamic executable, which i failed to check.)
also see "why does libtool strip link flags...": https://www.gnu.org/software/libtool/manual/html_node/Stripped-link-flags.html

it seems the properest(TM) way to do this is to do leave the static-ness to libtool (like it is apparently meant to be - libtool doesn't like the user messing with linker flags) and do the following:

```
./configure LDFLAGS="-L$PWD/../json-c" CFLAGS="-I./../json-c" --with-deadline"
AM_LDFLAGS="-all-static" make
```

going to push the fixed version, please let me know your thoughts.